pinot-controller/src/main/java/org/apache/pinot/controller/api/access/AccessControlUtils.java

```diff
@@ -43,55 +48,41 @@ public class AccessControlUtils {
    * @param endpointUrl the request url for which this access control is called
    * @param accessControl AccessControl object which does the actual validation
    */
-  public void validatePermission(String tableName, AccessType accessType, HttpHeaders httpHeaders, String endpointUrl,
-      AccessControl accessControl) {
-    validatePermission(Optional.of(tableName), accessType, httpHeaders, endpointUrl, accessControl);
-  }
+  public static void validatePermission(@Nullable String tableName, AccessType accessType,
+      @Nullable HttpHeaders httpHeaders, @Nullable String endpointUrl, AccessControl accessControl) {
+    String accessTypeToEndpointMsg =
+        String.format("access type '%s' to the endpoint '%s' for table '%s'", accessType, endpointUrl, tableName);
```

Comment:
If tableName is null, the log message doesn't look good. Previous message was handling this case better IMO.

Reply:
updated